api/krusty/remoteload_test.go

```diff
@@ -74,21 +78,46 @@ func testRemoteResource(require *require.Assertions, test *remoteResourceCase) {
 	require.NoError(fSys.RemoveAll(tmpDir.String()))
 }
 
+// isLocalEnv checks whether we can ascertain that tests are being run locally, instead of on the server
```

Comment:
nit: this is a little bit misleading, we're not actually checking that we're running locally vs we're on the server; we're checking specifically if `IS_LOCAL` is set (which we _assume_ determines whether we're running locally or not, but that may not always be the case)

Suggest specifically calling out that we're using the IS_LOCAL environment variable to help us here

Reply:
I see that the comment is misleading now. I'll move the comment into the function so that the comment isn't a repeat of the code and because the function is private.